src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -382,7 +1116,19 @@ public ResultSet getSchemas(String catalog, String schemaPattern) throws SQLExce
 
     @Override
     public ResultSet getClientInfoProperties() throws SQLException {
-        throw new SQLFeatureNotSupportedException("TODO");
+        ArrayList<BsonDocument> docs = new ArrayList<>();
+
+        MongoJsonSchema schema = MongoJsonSchema.createEmptyObjectSchema();
+        schema.addRequiredScalarKeys(
+                new Pair<>(NAME, BSON_STRING_TYPE_NAME),
+                new Pair<>(MAX_LEN, BSON_STRING_TYPE_NAME),
+                new Pair<>(DEFAULT_VALUE, BSON_STRING_TYPE_NAME),
+                new Pair<>(DESCRIPTION, BSON_STRING_TYPE_NAME));
+
+        // All fields in this result set are nested under the bottom namespace.
+        MongoJsonSchema botSchema = MongoJsonSchema.createEmptyObjectSchema();
+        botSchema.properties.put(BOT_NAME, schema);
+        return new MongoSQLResultSet(null, BsonExplicitCursor.EMPTY_CURSOR, botSchema);
```

Comment:
@nbagnard informed me that the clientInfoProperties are not for connection settings.  Postgres for example only returns applicationName for the session, mysql returns empty set.  So removed the connection settings and passing the empty cursor instead.  
I will keep the mysql clientInfoProperties as is to not change functionality.